### PR TITLE
Installer creates a forum and topic

### PR DESCRIPTION
**Pull request by browning**
Hey guys,
I took a stab at handling issue #91, making the installer create a default forum & topic.
Let me know if you have any suggestions.

The one thing thats kinda weird about this is that I am just using the first user in the db as the user that creates the topic & forum, I think its better than creating a new user in the db specifically for this, but I'm open to other approaches.

thanks

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+user = Forem.user_class.first
```
Comment (radar) [inline db/seeds.rb:1]: Forem.user_class should be a string, rather than a class. You will need to call constantize here.

File: db/seeds.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+unless user.nil?
+  forum = Forem::Forum.find_or_create_by_title( :category_id => Forem::Category.first.id,
```
Comment (radar) [inline db/seeds.rb:3]: Is there always a category created during the install?

Comment (browning) [inline db/seeds.rb:3]: Yes , as long as migrations are run, the general category gets created in the create_forem_categories migration.
Note that the seeds only run in the installer if the migrations are also run.

File: db/seeds.rb (hunk 1 above)
Comment (browning) [inline db/seeds.rb:1]: the constantize gets called automatically when you use user_class in lib/forem.rb right now

Comment (radar) [inline db/seeds.rb:1]: Ah, you're right.

**Comment by radar** [comment]
Merged, thank you! :)

**Pull request closed without merging**